pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -145,7 +147,23 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{}); err != nil {
+	service, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	podNetworkServicePollerDep.Spec.Replicas = &numNodes
+	podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Env = []corev1.EnvVar{
+		corev1.EnvVar{
+			Name:  "SERVICE_CLUSTER_IP",
+			Value: service.Spec.ClusterIP,
```

Comment:
This seems fine, but use your pod spec to pass this as an arg.  See example here: https://github.com/openshift/origin/blob/master/pkg/monitortests/network/disruptionpodnetwork/pod-network-poller-deployment.yaml#L32

Reply:
done